addons/full-signature/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Show full areas",
-  "description": "Shows the full signature of users on the forums, and shows full What I've been doing on profiles.",
+  "description": "Shows the full signature of users on the forums, stops Scratchblocks from being cut off at the edge of the screen, and shows full What I've been doing on profiles.",
```

Comment:
Not sure on what we should do with this description. We never updated it to reflect we support scroll for "What's happening?". Listing every feature in the description is not very useful

Reply:
Maybe something like *Makes some elements with limited height resizable or scrollable, such as user signatures on the forums and What I've been doing on profiles.*

Not really related, but should it be "on the forums" or "in the forums"? Different addons don't seem to agree about this.

Reply:
done